.gitignore

```diff
@@ -1,5 +1,6 @@
 /node_modules
-package-lock.json
+/tmp
```

Comment:
Is it something like .DS_Store?

Reply:
It is my own convention for a place to put temporary files specific to this project. It is like a local `/tmp`

Reply:
Ok.